runtime/gc_base/ObjectAccessBarrier.cpp

```diff
@@ -109,6 +109,95 @@ MM_ObjectAccessBarrier::tearDown(MM_EnvironmentBase *env)
 {
 }
 
+void
+MM_ObjectAccessBarrier::copyArrayCritical(J9VMThread *vmThread, GC_ArrayObjectModel *indexableObjectModel,
+	J9InternalVMFunctions *functions, void **data, J9IndexableObject *arrayObject, jboolean *isCopy)
```

Comment:
minor, i'd probably prefer not passing indexableObjectModel, but rather fetch it from _ext again
@dmitripivkine, opinion?

Reply:
I agree, let's get it from _ext